R/write_validation_report.R

```diff
@@ -0,0 +1,92 @@
+#' Create validation report from template
+#' @param pkg_name name of package
+#' @param template what validation report template from {valtools} to use,
+#' passed to \code{usethis::use_template}
+#' @param open boolean whether to open the validation report, passed to \code{usetthis::use_template}
+#' @importFrom usethis use_template proj_set
+#' @importFrom tools file_ext
+#' @importFrom rlang with_interactive
+#' @export
+vt_use_report <- function(pkg_name = desc::desc_get_field("Package"),
+                          template = "validation_report.Rmd",
+                          open = FALSE){
+
+  tryCatch({
+    dir <- vt_find_config()
+  },
+  error = function(e){
+    vt_use_config(pkg = vt_path())
+    dir <- vt_find_config()
+  })
```

Comment:
I think we should throw an error rather that silently adding the config. Also `vt_path()` is reliant on `vt_find_config()`, so the logic is circular. Instead just throw an error that the person needs to set up the config.